Vipul Kumar/if else/04_bonus.cpp

```diff
@@ -0,0 +1,24 @@
+#include<iostream>
+using namespace std;
+
+int main()
+{
+int year_of_service;
+int salary;
+int total_bonus;
+cin>>salary;
+cin>>year_of_service;
+if (year_of_service>5)
+{
+    total_bonus=(salary*5)/100;
```

Comment:
It will not give you right bonus, let salary = 50, then total_bonus = 50*5/100 = 250/100 = 2, you have taken it as int but right answer is 2.5, you have to take float total_bonus and total_bonus = salary*0.05;